test/e2e/cmd/run/run.go

```diff
@@ -220,6 +246,16 @@ func (h *helper) deployNamespaceOperator() error {
 	return h.kubectlApplyTemplateWithCleanup("config/e2e/namespace_operator.yaml", h.testContext)
 }
 
+func (h *helper) deployFilebeat() error {
+	if h.monitoringSecrets == "" {
+		log.Info("Not deploying filebeat")
```

Comment:
nit: maybe explain why it is not deployed ? 
```suggestion
		log.Info("No monitoring secrets provided, filebeat is not deployed")
```